main.css

```diff
@@ -1,14 +1,193 @@
 @import url('https://vaspacx-components.netlify.app/style.css');
+@import url('https://vaspacx-login.netlify.app/login.css');
 
-:root{
-    --primary-color:#3A7CE3;
-    --secondary-color:#090332;
-    --bg-color:#000000;
-    --font-color:#FFFFFF;
-    --text-color:#303338;
- }
 
  body{
-     background-color:var(--bg-color);
+     background-color:black;
      margin:0;
- }
\ No newline at end of file
+     display:block;
+ }
+
+ /* contact nav */
+
+ .contact-nav{
+    display: flex;
+    justify-content: space-between;
+    align-items:center;
+    padding: 7px 179px 8px;
+    border-bottom: 1px solid var(--text-color);
+}
+
+
+ .icon-button{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: transparent;
+    border: none;
+    gap: 9px;
+ }
+
+ .contact-nav-btn{
+    display: flex;
+    gap: 20px;
+ }
+
+ /* classes for all */
+
+ .bold-grey-text{
+    color: var(--text-color);
+    font-weight: 600;
+ }
+
+.bold-white-txt{
+    color:var(--font-color);
+
+}
+.margin-right-5px{
+margin-right:5px;
+}
+
+
+
+ul{
+    list-style: none;
+    margin-top: 417px;
+    cursor: pointer;
+}
+
+ul li{
+    border: 1px solid var(--text-color);
+    padding: 0px 1.3rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 17rem;
+    height: 50px;
+}
+
+li:hover{
+    color:var(--text-color);
+}
+.bg-primary{
+    background-color:var(--primary-color);
+}
+
+ /* logo */
+ .vaspacx{
+    font-family: 'Roboto', sans-serif;
+    font-size: 28px;
+    letter-spacing: 2px;
+    padding-bottom: 23px;
+    background: -webkit-linear-gradient(
+349deg
+,white 25%, #3a7ce3 29%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+
+/* search nav */
+
+.search-nav{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    border-bottom: 1px solid var(--text-color);
+    /* height: 57px; */
+    /* padding: 27px 0px; */
+    height: 100px;
+}
+
+.search-nav .search-cont{
+    display: flex;
+    gap: 20px;
+    align-items: center;
+}
+.search-nav select{
+    background-color: transparent;
+    border: 1px solid var(--text-color);
+    padding: 8px 8px;
```

Comment:
Don't use two values in shorthand if they are the same

Reply:
yes sure